integrationtest-service-framework/src/main/java/org/hypertrace/core/serviceframework/IntegrationTestServerUtil.java

```diff
@@ -40,6 +40,22 @@ public static void startServices(String[] services) {
     }
   }
 
+  public static void startServices(String testName, String[] services) {
```

Comment:
nit - dupe logic, can pass through 27 into this one as long as support null test name.

Reply:
Done